_data/simple-icons.json

```diff
@@ -9343,12 +9343,12 @@
         {
             "title": "Thunderbird",
             "hex": "0A84FF",
-            "source": "https://demo.identihub.co/thunderbird"
+            "source": "https://www.thunderbird.net/en-US/"
```

Comment:
This could also be https://github.com/thundernest/thunderbird-website/blob/d7446f3eee14b38f02ee60da7d4b4fb8c9ef20e3/media/svg/logo.svg

Reply:
ok, sure.